### PR TITLE
Bumped zio-http and caliban

### PR DESCRIPTION
**Pull request by SvenW**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -78,8 +78,10 @@ lazy val manager = project
   .settings(
     libraryDependencies ++=
       Seq(
-        "com.github.ghostdogpr" %% "caliban"          % calibanVersion,
-        "com.github.ghostdogpr" %% "caliban-zio-http" % calibanVersion
+        "com.github.ghostdogpr"       %% "caliban"          % calibanVersion,
+        "com.github.ghostdogpr"       %% "caliban-zio-http" % calibanVersion,
+        "com.softwaremill.sttp.tapir" %% "tapir-json-zio"   % "1.2.11",
+        "org.typelevel"               %% "cats-core"        % "2.9.0" % Test
```
Comment (ghostdogpr) [inline build.sbt]: Why was this needed?

Comment (SvenW) [reply build.sbt]: I think cats was included via zio-http in Caliban before but that is no longer the case so the end to end tests failed because it couldn't find cats any longer

Comment (ghostdogpr) [reply build.sbt]: Ah it's because I used `===` in the tests. I don't think that's worth depending on cats.
I think zio-prelude supports the same thing and it's included in the latest caliban. Can you try replacing the cats import by `import zio.prelude._`?

Comment (SvenW) [reply build.sbt]: Absolutely. I'll fix that as soon as possible

Comment (ghostdogpr) [reply build.sbt]: Actually I think `assertTrue` has some smart macro logic when using `==` and I don't think it gives as nice test failure messages with the triple equal. How about just using `==`?

**Review by ghostdogpr — APPROVED** [review]
Thanks!
